sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/StorageApiDynamicDestinations.java

```diff
@@ -35,6 +35,8 @@
 
     StorageApiWritePayload toMessage(T element) throws Exception;
 
+    StorageApiWritePayload toMessage(TableRow tableRow, boolean respectRequired) throws Exception;
```

Comment:
How is this relevant to this change?

Reply:
We can't simply take the unknownFields and convert them to a proto, as there may be missing required fields (because those fields are in the original proto). We need a way to do the conversion without enforcing nullability.